sql/core/src/test/resources/tpcds-plan-stability/approved-plans-v1_4/q4/explain.txt

```diff
@@ -362,267 +362,267 @@ Output [11]: [customer_id#25, year_total#26, customer_id#50, customer_first_name
 Input [12]: [customer_id#25, year_total#26, customer_id#50, customer_first_name#51, customer_last_name#52, customer_preferred_cust_flag#53, customer_birth_country#54, customer_login#55, customer_email_address#56, year_total#57, customer_id#84, year_total#85]
 
 (54) Scan parquet default.customer
-Output [8]: [c_customer_sk#87, c_customer_id#88, c_first_name#89, c_last_name#90, c_preferred_cust_flag#91, c_birth_country#92, c_login#93, c_email_address#58]
+Output [8]: [c_customer_sk#87, c_customer_id#88, c_first_name#89, c_last_name#90, c_preferred_cust_flag#91, c_birth_country#92, c_login#93, c_email_address#94]
```

Comment:
Seems cosmetic change only in the explain string?

Reply:
Yes and there is no plan change